vtex/faststore
# Fix: LH Unused CSS + Organizes CMS Sections and components styles.

## What's the purpose of this pull request?

- [x] Enables `SlideOver`/`Modal` components to be used as Sections (`CartSidebar`, `NavbarSlider`, `FilterSlider`,`RegionModal`).
- [x] Organizes `Sections`/`Common` components into Layout and Pages.
- [x] Uses CSS Modules per Sections (3 styles customizations levels, see video in slack channel).
- [x] Removes CSS Modules from other components and move styles to `@faststore/ui`.
- [x] Adjusts styles to use the same pattern used in the Atomic Design Epic approach in `@faststore/ui`.
- [x] Removes some stories (Storybook).  
- [x] Fix Unused CSS Problem for Homepage, PDP, PLP. 
- [x] Fix conflicts.

## How it works?

It prepares the repo to uses the CMS Section approach. It also enables the CSS Modules per section instead of importing all styles globally as before. Finally it fixes the Unused CSS Problem for Homepage, PDP, PLP, Search etc.


## How to test it?

All the pages should keep the same visually in Desktop and Mobile versions. You can see the coverage tab in devtools and check the unused CSS compared to previous version using the starter PR.

Note that the unused CSS problem is gone in the LH Check from the starter PR (But there are others LH Issues yet).

### Starters Deploy Preview

https://github.com/vtex-sites/starter.store/pull/44
